core/src/test/java/apoc/load/LoadJsonTest.java

```diff
@@ -183,13 +183,15 @@ public void testLoadJsonWithPathOptions() throws Exception {
                     assertEquals(map("foo",asList(1L,2L,3L)), row.get("value"));
                 });
     }
-    @Test @Ignore public void testLoadJsonGraphCommons() throws Exception {
+    @Test public void testLoadJsonGraphCommons() throws Exception {
 		String url = "https://graphcommons.com/graphs/8da5327d-7829-4dfe-b60b-4c0bda956b2a.json";
 		testCall(db, "CALL apoc.load.json($url)",map("url", url), // 'file:map.json' YIELD value RETURN value
```

Comment:
maybe remove the comment after this?

Reply:
Just thought it could be nice to leave it a bit cleaner 🤷‍♀️ no worries though :)